docs/Write-Plugin.md

```diff
@@ -0,0 +1,58 @@
+<h1 align="center">Fastify</h1>
+
+# How to write a good plugin
+First, thank you for deciding to write a plugin for Fastify. Fastify is a *batteries not included* framework and plugins are its strength, so thank you.<br>
```

Comment:
I would not define Fastify as "batteries not included" here. IMHO, those are the framework without a router/a logger etc.

Reply:
I have to disagree here having a router and a logger do not come close to making a framework batteries included. AFAIK the term comes from Djangos advertising and they included a very long list of features not included here for example templating, multi language, orm, default admin. Django, Spring, Rails, Sails, Laravel, etc are batteries included. Fastify, hapi, express, etc are not.

Reply:
Yes, I think it is a valid assertion to claim "batteries not included". Hapi is much more of a "batteries included" framework than Fastify. But I can be convinced otherwise as well. So whatever the consensus decides is fine with me.

Reply:
What about reword as "minimal"?